gateway/README.md

```diff
@@ -165,4 +276,4 @@ Find the Pod created with the `deployment.yaml` file and copy its name. Run this
 kc get pod {pod-name} -n $KYMA_EXAMPLE_NS -o json | jq '.spec.containers[].name'
```

Comment:
Because there is a label on a pod, we can use it to simplify this **Upstream connect error or disconnect/reset before headers:** troubleshooting to only one line:
`kubectl get pods -l example=gateway-service -n $KYMA_EXAMPLE_NS -o json | jq '.items[].spec.containers[].name'`
Also, we should not use `kc` in the example 😄

Reply:
That's not even mine! Fixing

Reply:
That is true. It is probably mine 😱But your PR is a great opportunity to have it finally fixed 😄